### PR TITLE
CNV-21974: clone in progress label

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

Forward also cloning DS and add label to them in the template list

Do you think other states will be added in the future? Maybe better to consider forwarding all the DS and filtering based on states later?  

## 🎥 Demo

**Before**
Screenshot from 2023-01-31 17-48-32

**After**
Screenshot from 2023-01-31 17-48-41

### CODE REVIEW

File: src/utils/resources/template/hooks/useVmTemplateSource/utils.ts [hunk 1]
```diff
@@ -108,6 +108,19 @@ export const getDataSourcePVC = (name: string, ns: string) =>
...
 
+/**
+ * a function that returns true if the data source is cloning in progress
+ * @param dataSource the data source to check if cloning
+ * @returns true if the data source is in cloning state, false otherwise
+ */
+export const isDataSourceCloning = (dataSource: V1beta1DataSource): boolean =>
+  dataSource?.status?.conditions?.some(
+    (c) =>
+      c.type === 'Ready' &&
```
Comment (hstastna) [inline src/utils/resources/template/hooks/useVmTemplateSource/utils.ts:119]: Do the conditions always have `type`, `status` and `reason` present?

**Review by hstastna — COMMENTED** [review]
Just had one question, otherwise looks good to me! 💪🏽

File: src/utils/resources/template/hooks/useVmTemplateSource/utils.ts (hunk 1 above)
Comment (upalatucci) [reply src/utils/resources/template/hooks/useVmTemplateSource/utils.ts:119]: `reason` is optional. I'll fix that. Thanks for the good catch

**Review by hstastna — APPROVED** [review]

**Pull request merged**